File: hackerrank.com/problem-solving/mini_max_sum.scala
// https://www.hackerrank.com/challenges/mini-max-sum/problem

import scala.io.StdIn._

object Solution {
  
  def miniMaxSum(arr: Array[Long]): Unit = {
    var ar = arr.sorted
    val min = ar.take(4).sum
    val max = ar.drop(1).sum
    println(min + " " + max)
  }
  
  def main(args: Array[String]) {

    val n = readLine.split(" ").map(_.trim.toLong)
    miniMaxSum(n)
  }
}

File: hackerrank.com/problem-solving/mini_max_sum.py
#  https://www.hackerrank.com/challenges/mini-max-sum/problem

def miniMaxSum(arr):
  ar = sorted(arr)
  print(sum(ar[0:4]), sum(ar[1:5]))


if __name__ == '__main__':
    arr = list(map(int, input().split()))
    miniMaxSum(arr)